app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt

```diff
@@ -105,14 +105,9 @@ class ConversationViewModel @Inject constructor(
 
     fun navigateToInitiatingCallScreen() {
         viewModelScope.launch {
-//            navigationManager.navigate(
-//                command = NavigationCommand(
-//                    destination = NavigationItem.OngoingCall.getRouteWithArgs()
-//                )
-//            )
             navigationManager.navigate(
                 command = NavigationCommand(
-                    destination = NavigationItem.IncomingCall.getRouteWithArgs()
+                    destination = NavigationItem.OngoingCall.getRouteWithArgs()
```

Comment:
Just changing back to what it was before.
Changed before only for testing purposes.